app/client/models/AdminChecks.ts

```diff
@@ -169,6 +169,16 @@ It is good practice not to run Grist as the root user.
   'reachable': {
     info: `
 The main page of Grist should be available.
+`
+  },
+
+  'websockets': {
+    info: `
+Websocket connections need HTTP 1.1 and the ability to pass a few
+extra headers in order to work. Sometimes a reverse proxy can
+interfere with these requirements. See <a
+href="https://support.getgrist.com/self-managed/#how-do-i-run-grist-on-a-server">this
+documentation</a> for more explanation.
```

Comment:
This isn't correct, but I just realised that grainjs is our own thing. Is there an easy way to put a link here? I'm not sure how to cross-reference otherwise.

Reply:
Maybe `makeLinks` from `links.ts`?

Reply:
That `ui2018` in the module path doesn't inspire confidence. Is that really the best way? 😕

Reply:
No, it's the easy way.

The best way would be to change the details to be `DomContents` instead of a string. `GristTooltips.ts` is a good example of largely static blocks of text comprised of DOM nodes.

It'll require a bit of refactoring since details are currently strings, but it shouldn't be much if you're up for it.

Reply:
I think it might be worth it, might try this tomorrow. I may like to add more links for other explanations here. Thanks!

Reply:
Can you defer that bit? AdminChecks changes in a parallel PR. Put small "add more explanation link to websocket probe" card in the kanban for now?

Reply:
Right-o, will do.